"can1": "hw_mrr"
"can2": "bcan"
"can3": "ccan"
"can4": "hw_f_lr"
"can5": "private"
"can6": "mcu_debug"
"can7": "private"
"can8": "hw_r_lr"

